content/src/policies/todoApp.POST.todos.rego

```diff
@@ -1,13 +1,25 @@
 package todoApp.POST.todos
 
-import input.user
-import future.keywords.in
-import data.todoApp.common.is_member_of
+# This policy determines whether the user can create todos
 
 default allowed = false
 
+# Only members of the "resource-creators" instance can create a new Todo.
+#
+# Use the "Check" middleware convention
```

Comment:
Are you leaving this module in place for backward compatibility?
If the app uses the check middleware, it's going to hit the `rebec.check` module.